livesync/indico_livesync/uploader.py

```diff
@@ -56,7 +54,7 @@ def run_initial(self, records, total):
             ((rec, SimpleChange.created) for rec in records),
             total,
             lambda entry: entry[0].id,
```

Comment:
It was me changing this earlier but I just saw we can switch to the same method
```suggestion
            lambda entry: getattr(entry[0], 'id'),
```

Reply:
uh why? getattr with a static attr and no default isn't really useful

Reply:
Never mind, I had the impression your intention was to make it more efficient while repeating this multiple times